examples/scripts/data_io/dataframe/validate_dataframe.py

```diff
@@ -0,0 +1,32 @@
+from tamr_toolbox.models.validation_check import ValidationCheck
+import pandas as pd
+from tamr_toolbox.data_io import dataframe
+
+
+def main() -> ValidationCheck:
+    """returns columns which failed checks
+    Returns: ValidationCheck object consisting of Pass/Fail and dict of failed columns
+    """
+
+    def ensure_not_2(value):
```

Comment:
do we want an example where we have nested function definitions?

Reply:
Do you mean a check function that checks multiple conditions with multiple functions?

Reply:
I just mean I worry about giving people sharp scissors in examples. To me this is slightly sharper scissor than just:

```
def ensure_not_2(value):
....

def main()
```

Reply:
also has the benefit of hinting that you can use any function you import from anywhere

Reply:
Ah, yeah.  It's probably better to have the function defined outside of `main`, so there are no functions defined inside of functions.

Reply:
@DIvkov575 Can you move this function outside of `main` here?  It's a minor change but I think will make the example clearer to users.